### PR TITLE
Changed Nav View Text if First Name is null

### LINKED ISSUE #700: Showing 'null' in Navigation View, if First Name not provided.
**Is it a ?**
- [x] Chore
    
### Chore

**Description**

Removing 'null' from Navigation View if First Name not provided

**Would you like to work on the issue?**

I would like to work on this issue.

### PR DESCRIPTION
**Pull request by YashishDua**
Fixes #700 

Changes: If First Name is null, email is shown instead. I used data binding and added dynamic code in XML only.

Screenshots for the change: 
screenshot_2018-03-08-15-23-17-1214175332

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/app/module/main/MainActivity.java [hunk 1]
```diff
@@ -59,11 +59,13 @@
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+
         OrgaApplication
             .getAppComponent()
             .inject(this);
 
         super.onCreate(savedInstanceState);
+
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/module/main/MainActivity.java]: Unneeded changes in this file

Comment (YashishDua) [reply app/src/main/java/org/fossasia/openevent/app/module/main/MainActivity.java]: So, I should just remove this line?

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/openevent/app/module/main/MainActivity.java]: And the one added above it

File: app/src/main/java/org/fossasia/openevent/app/module/main/MainActivity.java [hunk 2]
```diff
@@ -59,10 +59,10 @@
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/module/main/MainActivity.java]: Still present

File: app/src/main/java/org/fossasia/openevent/app/module/main/MainActivity.java [hunk 3]
```diff
@@ -59,10 +59,10 @@
...
         OrgaApplication
             .getAppComponent()
             .inject(this);
-
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/module/main/MainActivity.java]: Unneeded extra change

**Pull request merged**